### PR TITLE
fix: Add proxy initializer to fetch infrastructural data [DHIS2-9451]

### PR DESCRIPTION
**Pull request by zubaira**

### CODE REVIEW

**Review by maikelarabori — APPROVED** [review]

**Review by luciano-fiandesio — COMMENTED** [review]
hi @zubaira which collection was throwing a Lazy exception?

File: dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/ConfigurationController.java [hunk 1]
```diff
@@ -203,9 +204,13 @@ public void removeOfflineOrganisationUnitLevel()
     }
 
     @RequestMapping( value = "/infrastructuralIndicators", method = RequestMethod.GET )
-    public @ResponseBody IndicatorGroup getInfrastructuralIndicators( Model model, HttpServletRequest request )
+    public @ResponseBody IndicatorGroup getInfrastructuralIndicators()
     {
-        return configurationService.getConfiguration().getInfrastructuralIndicators();
+        IndicatorGroup group = configurationService.getConfiguration().getInfrastructuralIndicators();
```
Comment (zubaira) [inline dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/ConfigurationController.java:209]: @luciano-fiandesio IndicatorGroup class has a list of indicators.

**Review by larshelge — CHANGES_REQUESTED** [review]
This looks too arbitrary to me. It seems like we have a lot of Hibernate proxy initialization issues around now and we are adding fixes all over. Would be nice to find a more centralized solution. We should not expose Hibernate in the Web API controller layer.

**Review by stian-sandvold — APPROVED** [review]

**Comment by netroms** [comment]
I have solved this problem without using Hibernate initialization, the core of the problem lies in the Hibernate5Module. I can't commit yet because @vietnguyen is working in parallel on fixing the issues with auditing that are also using the Hibernate5Module. So until then I think we should close this PR.

**Pull request closed without merging**